numpy/f2py/rules.py

```diff
@@ -1048,49 +1089,62 @@
         'keys_capi': {isoptional: ',&#varname#_capi'},
         '_check': l_and(isarray, isintent_nothide)
     }, {
-        'frompyobj': ['\t#setdims#;',
-                      '\tcapi_#varname#_intent |= #intent#;',
-                      {isintent_hide:
-                       '\tcapi_#varname#_tmp = array_from_pyobj(#atype#,#varname#_Dims,#varname#_Rank,capi_#varname#_intent,Py_None);'},
-                      {isintent_nothide:
-                       '\tcapi_#varname#_tmp = array_from_pyobj(#atype#,#varname#_Dims,#varname#_Rank,capi_#varname#_intent,#varname#_capi);'},
-                      """\
-\tif (capi_#varname#_tmp == NULL) {
-\t\tPyObject *exc, *val, *tb;
-\t\tPyErr_Fetch(&exc, &val, &tb);
-\t\tPyErr_SetString(exc ? exc : #modulename#_error,\"failed in converting #nth# `#varname#\' of #pyname# to C/Fortran array\" );
-\t\tnpy_PyErr_ChainExceptionsCause(exc, val, tb);
-\t} else {
-\t\t#varname# = (#ctype# *)(PyArray_DATA(capi_#varname#_tmp));
+        'frompyobj': [
+            '\t#setdims#;',
+            '\tcapi_#varname#_intent |= #intent#;',
+            '    const char * capi_errmess = "#modulename#.#pyname#:'
```

Comment:
LGTM suggests this line end in a ',' for consistency

Reply:
yeah, but that would be wrong: the two strings should be concatenated so that in the final result they would appear in a single line. Ending the line with a comma would join the two strings with `\n` and a compiler error would follow due to C's literal string being multi-lined...

Is there a way to let LGTM ignore such cases?

Reply:
maybe just a comment 
```suggestion
            # line split intentionally for linting
            '    const char * capi_errmess = "#modulename#.#pyname#:'
```

Reply:
this, or do the actual string concatenate:
```python
  ('    const char * capi_errmess = "#modulename#.#pyname#:'
   + '....'),
```
?

Btw, there are many places in this file where lines need to split similarly for linting, and having those comment lines for the sake of LGTM or flake, sounds annoying.

Reply:
Adding parenthesis was sufficient to fix the LGTM warning.